oauth2client/file.py

```diff
@@ -20,17 +20,28 @@
 
 import os
 import threading
+import warnings
 
 from oauth2client import client
 
 
 __author__ = 'jcgregorio@google.com (Joe Gregorio)'
 
+_SYM_LINK_MESSAGE = 'File: {0}: Is a symbolic link.'
+
+_IS_DIR_MESSAGE = '{0}: Is a directory'
+
+_MISSING_FILE_MESSAGE = 'Cannot access {0}: No such file or directory'
+
 
 class CredentialsFileSymbolicLinkError(Exception):
     """Credentials files must not be symbolic links."""
 
 
+class CredentialsFileDirectoryError(Exception):
```

Comment:
I greatly prefer not raising an exception for `get()` not finding credentials. Based on the usage of this library, that is not an error. The usage pattern mostly seen is:

```
credentials = storage.get()
if not credentials():
    # obtain them somehow.
    storage.put(credentials)
```